src/main/java/com/backend/blooming/goal/presentation/dto/request/UpdateGoalRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.backend.blooming.goal.presentation.dto.request;
+
+import jakarta.validation.constraints.FutureOrPresent;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public record UpdateGoalRequest(
+        
+        String name,
+        String memo,
+        
+        @DateTimeFormat(pattern = "yyyy-MM-dd")
+        @FutureOrPresent
```

Comment:
`@FutureOrPreset` 조건에 어긋나면 예외가 발생할 것입니다.
그런데 현재는 예외가 발생했을 때 예외 메시지가 따로 없어 어떤 문제인지 확인이 어려울 수도 있을 것 같아요!
아래와 같이 예외 메시지를 추가해 주면 어떨까요?
```suggestion
        @FutureOrPresent(message = "~")
```

Reply:
좋습니다! 예외메시지 남기도록 하겠습니다!

Reply:
`endDate` `null`값 request에 대해 더 말씀드리자면, 테스트 코드상으로는 `@DateTimeFormat` 및 `@FutureOrPresent`에서도 예외가 발생하지 않고 null 값을 그대로 받아올 수 있었습니다. 실제로 클라이언트와 통신을 했을 때도 오류가 안 나는지는 서버에 올려보고 테스트 해봐야 알 수 있을 것 같습니다.